tests/financial/test_ny_stock_exchange.py

```
@@ -282,109 +338,412 @@ def test_christmas_day(self):
         self.assertHolidayName(f"{name} (observed)", obs_dts)
         self.assertNoNonObservedHoliday(obs_dts)
+        no_obs_dts = (
+            "1943-12-24",
+            "1948-12-24",
+        )
+        self.assertNoHolidayName(name_observed, no_obs_dts)
+        self.assertNoHoliday(no_obs_dts)
 
-    def test_special_holidays(self):
-        # add to this list as new historical holidays are added
+    def _make_special_holiday_list(self, begin, end, *, days=None):
```

Ruff (0.14.7)

Missing return type annotation for private function `_make_special_holiday_list`

(ANN202)